wikipediapreview.php

```diff
@@ -13,7 +13,7 @@
  * License URI: https://github.com/wikimedia/wikipedia-preview/blob/main/LICENSE
  */
 
-DEFINE( 'WIKIPEDIA_PREVIEW_PLUGIN_VERSION', '1.14.3' );
+DEFINE( 'WIKIPEDIA_PREVIEW_PLUGIN_VERSION', '1.15.0' );
```

Comment:
You can revert/remove this change, as we will handle it as part of the upcoming version deployment.